src/main/java/uk/gov/hmcts/reform/sscs/evidenceshare/callback/handlers/DwpUploadResponseHandler.java

```diff
@@ -67,24 +68,36 @@ public void handle(CallbackType callbackType, Callback<SscsCaseData> callback) {
 
     private void handleChildSupportAndSscs5Case(Callback<SscsCaseData> callback) {
         if (StringUtils.equalsIgnoreCase(callback.getCaseDetails().getCaseData().getDwpFurtherInfo(), "Yes")) {
-            SscsCaseData caseData = setDwpState(callback);
-            caseData.setInterlocReviewState(AWAITING_ADMIN_ACTION);
-            updateEventDetails(caseData, callback.getCaseDetails().getId(),
-                EventType.DWP_RESPOND, "Response received", "Update to response received as an Admin has to review the case");
+            updateEventDetails(callback.getCaseDetails().getId(),
+                EventType.DWP_RESPOND, "Response received", "Update to response received as an Admin has to review the case",
+                    sscsCaseData -> {
+                        sscsCaseData.setDwpState(RESPONSE_SUBMITTED_DWP);
+                        sscsCaseData.setInterlocReviewState(AWAITING_ADMIN_ACTION);
+                        log(callback, EventType.DWP_RESPOND);
+                    });
         } else if (StringUtils.equalsIgnoreCase(callback.getCaseDetails().getCaseData().getDwpFurtherInfo(), "No")) {
             if (isBenefitTypeSscs5(callback.getCaseDetails().getCaseData().getBenefitType()) && !StringUtils.equalsIgnoreCase(callback.getCaseDetails().getCaseData().getDwpEditedEvidenceReason(), "phme")) {
                 triggerReadyToListEvent(callback);
             } else {
-                SscsCaseData caseData = setDwpState(callback);
-                caseData.setInterlocReviewState(REVIEW_BY_JUDGE);
                 if (isBenefitTypeSscs5(callback.getCaseDetails().getCaseData().getBenefitType())) {
-                    updateEventDetails(caseData, callback.getCaseDetails().getId(),
-                        EventType.DWP_RESPOND, "Response received", "Update to response received as an Admin has to review the case");
+                    updateEventDetails(callback.getCaseDetails().getId(),
+                        EventType.DWP_RESPOND, "Response received", "Update to response received as an Admin has to review the case",
+                            sscsCaseData -> {
+                                sscsCaseData.setDwpState(RESPONSE_SUBMITTED_DWP);
+                                sscsCaseData.setInterlocReviewState(REVIEW_BY_JUDGE);
+                                log(callback, EventType.DWP_RESPOND);
+                            });
                 }
             }
         }
     }
 
+    private static void log(Callback<SscsCaseData> callback, EventType eventType) {
```

Comment:
don't need to extract this method you can just log it :)

Reply:
Removed log method